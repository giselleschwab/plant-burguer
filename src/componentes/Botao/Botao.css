.botao {
  width: 60%;
  height: 4rem;
  background-color: #D65D1A;
  opacity: 80%;
  box-shadow: 1px 2px 0px #000000;
  border-radius: 10px;
  margin: auto;
  color: #FFFFFF;
  font-family: Helvetica, Arial, sans-serif;
  font-size: 1.2rem;
  font-weight: bold;
  cursor: pointer;
}

.botao:hover{
  background-color: #435C3D;
}