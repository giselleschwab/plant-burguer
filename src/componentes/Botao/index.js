import './Botao.css';

function Botao(props) {
  return (
    <button type="button" className="botao" onClick={props.onClick}>
      {props.children}
    </button>
  );
}

export default Botao;
