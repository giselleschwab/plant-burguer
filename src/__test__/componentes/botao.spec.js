import { render, screen } from "@testing-library/react";
import Botao from "../../componentes/Botao";

it("Deve renderizar o botÃ£o corretamente", () => {
    render(
        <Botao > ENTRAR </Botao>
    )
    const btn = screen.getByText("ENTRAR");
    expect(btn).toBeInTheDocument();
    expect(btn).toMatchSnapshot();
})