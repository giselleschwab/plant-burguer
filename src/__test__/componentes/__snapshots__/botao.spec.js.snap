// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deve renderizar o bot√£o corretamente 1`] = `
<button
  class="botao"
>
   ENTRAR 
</button>
`;
